src/GtkUtilities.jl

```diff
@@ -44,7 +44,7 @@ include("panzoom.jl")
 using .PanZoom
 import .PanZoom: interior, fullview   # for extensions
 
-function Base.copy!{C<:Colorant}(ctx::CairoContext, img::AbstractArray{C})
+function Base.copy!{C<:Union{Colorant,Number}}(ctx::CairoContext, img::AbstractArray{C})
```

Comment:
I'm aware of that. But you seem to regard packages as having sacrosanct boundaries. What if I say that it's even more fine-grained, at the module level? We use modules for encapsulation all the time and don't worry a bit about that kind of "piracy."

When people disagree about the scope of a package, there are two solutions:
- `using Colors --WithoutMathematics` or `using Colors --WithMathematics`. Of course these don't work yet, and most obvious ways to fake this would break precompilation. And of course it introduces the same problem you're worried about anyway, because it would depend exactly on how the package was loaded.
- split the package into two. Many of us will consider the "extra functionality" one the "true" version of the package. 

The point is that packages *aren't* an inviolable level of granularity. Not with julia's current design, at least.  Piracy at some level is built into Julia's entire philosophy: if I write modules/packages `NumberTypes1` and `NumberTypes2` and then want to be able to add `x::NumberType1 + y::NumberType2` then I'm going to engage in some type piracy. Neither package should depend on the other, and it's Base that owns `+`. There *isn't* any reasonable solution for this except for type piracy, and it would be very un-Julian to insist otherwise.

Problems of dependency management do not hold a card that trumps all other issues of design.